server/config/config.go

```
@@ -573,37 +569,23 @@ func (c *PDServerConfig) adjust(meta *configutil.ConfigMetaData) error {
 	} else if c.GCTunerThreshold > maxGCTunerThreshold {
 		c.GCTunerThreshold = maxGCTunerThreshold
 	}
-	if err := c.migrateConfigurationFromFile(meta); err != nil {
+	if err := migrateConfigurationFromFile(meta); err != nil {
 		return err
 	}
 	return c.Validate()
 }
 
-func (c *PDServerConfig) migrateConfigurationFromFile(meta *configutil.ConfigMetaData) error {
+func migrateConfigurationFromFile(meta *configutil.ConfigMetaData) error {
 	oldName, newName := "trace-region-flow", "flow-round-by-digit"
-	defineOld, defineNew := meta.IsDefined(oldName), meta.IsDefined(newName)
+	defineOld, _ := meta.IsDefined(oldName), meta.IsDefined(newName)
 	switch {
-	case defineOld && defineNew:
-		if c.TraceRegionFlow && (c.FlowRoundByDigit == defaultFlowRoundByDigit) {
-			return errors.Errorf("config item %s and %s(deprecated) are conflict", newName, oldName)
-		}
-	case defineOld && !defineNew:
-		if !c.TraceRegionFlow {
-			c.FlowRoundByDigit = math.MaxInt8
-		}
+	case defineOld:
+		return errors.Errorf("config item %s and %s(deprecated) are conflict", newName, oldName)
```

Do we need to report an error? How about print a warning?